### PR TITLE
Added single-sample workflows

### PR DESCRIPTION
**Pull request by dvismer-technome**
Added paired-fastq-to-unmapped-bam and PairedEndSingleSampleWf workflows to wgs

### CODE REVIEW

File: wgs/PairedEndSingleSampleWf/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+The workflow in this repository is the Paired End Single Sample workflow provided by the Broad Institute.
```
Comment (mbookman) [inline wgs/PairedEndSingleSampleWf/README.md]: > The workflow in this directory implements alignment and variant calling according to the GATK Best Practices for germline SNP and Indel discovery in human whole-genome sequencing (WGS) data.

File: wgs/PairedEndSingleSampleWf/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+A copy of the PairedEndSingleSampleWf.wdl file was made on January 4, 2018 from the Broad's github repo:
```
Comment (mbookman) [inline wgs/PairedEndSingleSampleWf/README.md]: > A copy of the January 4, 2018 version of the `PairedEndSingleSampleWf.wdl` file was made from the Broad Institute's github repo:

(I'm not actually sure when we made the copy - just know it was done after 1/4/2018).

File: wgs/PairedEndSingleSampleWf/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+https://github.com/gatk-workflows/broad-prod-wgs-germline-snps-indels/blob/7cc39c1e31b0b8fc5fd4f0e58180c0edea8d1246/PairedEndSingleSampleWf.wdl
+
+No changes were required to the workflow for use in AMP PD.
```
Comment (mbookman) [inline wgs/PairedEndSingleSampleWf/README.md]: > No changes were made to the workflow for AMP PD WGS single sample processing.

File: wgs/paired-fastq-to-unmapped-bam/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+The workflow in this repository is the Paired FastQ to Unmapped BAM workflow provided by the Broad Institute.
```
Comment (mbookman) [inline wgs/paired-fastq-to-unmapped-bam/README.md]: > The workflow in this repository converts paired FASTQ to uBAM and adds read group information.

File: wgs/paired-fastq-to-unmapped-bam/README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+
+A copy of the paired-fastq-to-unmapped-bam.wdl file was made on February 23, 2018 from the Broad's github repo:
```
Comment (mbookman) [inline wgs/paired-fastq-to-unmapped-bam/README.md]: A copy of the February 23, 2018 version of the `paired-fastq-to-unmapped-bam.wdl` file was made from the Broad Institute's github repo:

File: wgs/paired-fastq-to-unmapped-bam/README.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+
+https://github.com/gatk-workflows/seq-format-conversion/blob/413fca8295d44ea0a6c147bab6663d7cfdd1d808/paired-fastq-to-unmapped-bam.wdl
+
+No changes were required to the workflow for use in AMP PD.
```
Comment (mbookman) [inline wgs/paired-fastq-to-unmapped-bam/README.md]: No changes were made to the workflow for AMP PD WGS single sample processing.

**Review by mbookman — CHANGES_REQUESTED** [review]
Just a few small updates to the README files.
Otherwise LGTM.

File: wgs/PairedEndSingleSampleWf/README.md (hunk 1 above)
Comment (dvismer-technome) [reply wgs/PairedEndSingleSampleWf/README.md]: This change has been made

File: wgs/PairedEndSingleSampleWf/README.md (hunk 2 above)
Comment (dvismer-technome) [reply wgs/PairedEndSingleSampleWf/README.md]: This change has been made

File: wgs/PairedEndSingleSampleWf/README.md (hunk 3 above)
Comment (dvismer-technome) [reply wgs/PairedEndSingleSampleWf/README.md]: This change has been made

File: wgs/paired-fastq-to-unmapped-bam/README.md (hunk 4 above)
Comment (dvismer-technome) [reply wgs/paired-fastq-to-unmapped-bam/README.md]: This change has been made

File: wgs/paired-fastq-to-unmapped-bam/README.md (hunk 5 above)
Comment (dvismer-technome) [reply wgs/paired-fastq-to-unmapped-bam/README.md]: This change has been made

File: wgs/paired-fastq-to-unmapped-bam/README.md (hunk 6 above)
Comment (dvismer-technome) [reply wgs/paired-fastq-to-unmapped-bam/README.md]: This change has been made

**Comment by mbookman** [comment]
Let's change the directory name from `PairedEndSingleSampleWf` to `wgs-germline-snps-indels`.

**Comment by dvismer-technome** [comment]
Will do.  Is the name of the WDL okay, or does that need to change as well?

 
Dave


David Vismer
Senior Engineer | technome.io
+1 (703) 477-6235
d.vismer@technome.io

-----Original message-----

**Comment by mbookman** [comment]
LGTM. When you merge, be sure to "Squash and merge" and clean up (compact) the commit comment.

**Pull request merged**